utils/uninstall.sh

```diff
@@ -110,7 +110,8 @@ remove_parsed() {
         IFS=$'\n'
         ask_remove $(parse_env)
         if [[ "$IPATH" =~ ".wasmedge" ]]; then
-            ask_remove $(find "$IPATH" -type d -empty -print)
+            ask_remove $(find "$IPATH" -depth -type d -empty -print)
+            ask_remove $(find "$IPATH" -depth -type d -empty -print)
```

Comment:
Do we need to remove it twice?

Reply:
It looks strange but the first time it searches `~/.wasmedge/include/wasmedge` is contained inside `~/.wasmedge/include` which is why it gets excluded and we need to find for empty directory one more time.

Reply:
Example:
![image](https://user-images.githubusercontent.com/61797109/149318074-170af4fa-a8d5-478a-8af4-0365549756ad.png)

Reply:
![image](https://user-images.githubusercontent.com/61797109/149318349-07572dc2-9127-40d8-a96f-ceae1839df56.png)

Reply:
Got it.